scripts/Modules/mkw_utils.py

```diff
@@ -48,8 +48,9 @@ def delta_position(playerIdx=0):
 
     return physics_ref.position() - dynamics_ref.position()
 
-"""Next 3 functions are used for exact finish display"""
-def getIGT(lap, player):
+# Next 3 functions are used for exact finish display
+
+def get_igt(lap, player):
```

Comment:
If we are going to enforce the snake case function rule, it should be done repository-wide in this PR.